app/javascript/gobierto_investments/webapp/components/MapTour.vue

```diff
@@ -0,0 +1,197 @@
+
+      this.subsetItems = this.items;
+
+      this.setGeoJSONs(this.items);
+    })
+  },
+  computed: {
+    center() {
+      return this.geojsons.length !== 0 ? Object.values(this.geojsons).reverse() : this.coordinatesCities;
+    },
+  },
+  mounted() {
+    setTimeout(() => { this.cardsOnScreen(this.activeCardId) }, 3000)
+  },
+  methods: {
+    onMapLoaded(event) {
+      this.map = event.map;
+      this.addMarkers
+    },
+    convertWKTtoGeoJSON(wktString) {
+      const wkt = new Wkt.Wkt();
+      wkt.read(wktString);
+      return wkt.toJson();
+    },
+    setGeoJSONs(items) {
+      this.geojsons = [];
+
+      // get the Well-Known Text (WKT)
+      const markerWithLocation = items.reduce((acc, i) => {
+        if (i.location && i.location !== null) {
+          acc.push(i);
+        }
+        return acc;
+      }, []);
+
+      // convert all WKT objets to GeoJSON, and add them to a feature group
+      for (let index = 0; index < markerWithLocation.length; index++) {
+        const { id, location, photo, title, description, attributes } = markerWithLocation[index];
+        this.geojsons.push({ ...this.convertWKTtoGeoJSON(location), id, photo, title, description, attributes });
+      }
+    },
+    cardsOnScreen(elementCard) {
+      this.activeCard(this.activeCardId);
+    },
+    randomNumbers(min, max) {
+      return Math.floor(Math.random() * (max - min + 1) + min);
+    },
+    activeCard(card) {
+      this.titleCard = this.geojsons[card].title.ca
+      this.photoCard = this.geojsons[card].photo
+      this.item = this.geojsons[card].id
+
+      const [lat, lng] = Object.values(this.geojsons[card].coordinates)
+
+      if (this.geojsons[card].coordinates.length <= 1) {
+        this.coordinatesMarker = this.geojsons[card].coordinates[0][0]
+        const [lat, lng] = Object.values(this.geojsons[card].coordinates[0][0])
+        this.map.flyTo({
+          center: [lat, lng],
+          zoom: this.randomNumbers(15, 17),
+          bearing: this.randomNumbers(-60, 60),
+          pitch: this.randomNumbers(10, 60),
+          speed: 0.75
+        });
+
+      } else {
+        this.coordinatesMarker = this.geojsons[card].coordinates
+        this.map.flyTo({
+          center: [lat, lng],
+          zoom: this.randomNumbers(15, 17),
+          bearing: this.randomNumbers(-60, 60),
+          pitch: this.randomNumbers(10, 60),
+          speed: 0.75
+        });
+
+      }
+      this.activeCardId += 1
+      if (this.activeCardId < this.geojsons.length) {
+        this.activeTour = setTimeout(() => { this.cardsOnScreen(this.activeCardId) }, 3000)
+        this.activeTour
+      }
+    },
+    reloadTour() {
+      clearTimeout(this.activeTour)
+      this.activeCardId = 0
+      this.cardsOnScreen(this.activeCardId)
+    },
+    backInvestments() {
+      this.$router.back({ name: "home" });
+    },
+    navTo(item) {
+      this.$router.push({ name: "project", params: { id: item } });
```

Comment:
Not sure if this is working well, if you visit a project (i.e http://mataro.gobierto.test/inversiones/proyectos/72) you see it with different styles that if you clik in the project card of the virtual tour.

Reply:
Done.